bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/BigtableSession.java

```diff
@@ -296,7 +296,11 @@ public Credentials call() throws IOException {
         (scheduledRetries == null) ? createDefaultRetryExecutor() : scheduledRetries;
 
     Builder<HeaderInterceptor> headerInterceptorBuilder = new ImmutableList.Builder<>();
-    headerInterceptorBuilder.add(new UserAgentInterceptor(options.getUserAgent()));
+
+    String javaVersion = "jdk" + System.getProperty("java.specification.version");
```

Comment:
set/update user agent here:
public static final String USER_AGENT = getUserAgent();
in  BigtableConstants.java ?